backend/services/monthly_dump_service.py

```
@@ -0,0 +1,274 @@
+        response = self.entry_controller.fetch_user_entries_in_range(
+            user_id=user_id,
+            start_utc_iso=start_utc.isoformat(),
+            end_utc_iso=end_utc.isoformat()
+        )
+        # Apply the is_private=False filter as per existing logic
+        entries = response.data or []
+        return [e for e in entries if not e.get("is_private", False)]
+
+    def build_monthly_dump(self, inputs: MonthlyDumpInputs) -> MonthlyDumpResult:
+        start_utc, end_utc = self.get_month_bounds(inputs.month, inputs.timezone)
+        logger.info(
```

Ruff (0.15.10)

[error] Too many arguments for `logging` format string

(PLE1205)